browser/brave_profile_prefs.cc

```diff
@@ -461,6 +461,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
                                 base::Value(true));
   registry->RegisterBooleanPref(kEnableWindowClosingConfirm, true);
   registry->RegisterBooleanPref(kEnableClosingLastTab, true);
+  registry->RegisterBooleanPref(kEnableShowFullscreenReminder, true);
```

Comment:
Maybe rename to `kEnableShowFullscreenExitInstructions` for clarity?

Reply:
Actually, I don't think we need the word `Enable` in there at all. `Show` is the action (we're either going to show the instructions or not).

Reply:
Done.

Renamed to `kShowFullscreenReminder`.


Do you want me to change to `kShowFullscreenExitInstruction`, If yes, I need to change the String in `.grd` as well, Kindly confirm.

Reply:
I think `kShowFullscreenExitInstructions` is a little clearer, but it's up to you.